```
throw(e)
```

将对象作为异常抛出。

另见：[`rethrow`](@ref)，[`error`](@ref)。
